site_scons/components/__init__.py

```diff
@@ -172,7 +172,19 @@ def define_mercury(reqs):
         MERCURY_DEBUG = '-DMERCURY_ENABLE_DEBUG=OFF '
     retriever = \
         GitRepoRetriever('https://github.com/mercury-hpc/mercury.git',
-                         True)
+	True)
+
+    reqs.define("UCX", libs=['ucp'])
+
+    if reqs.check_component('UCX'):
+        UCX = '-DNA_USE_UCX=ON ' \
+        '-DUCX_INCLUDE_DIR=/usr/include '\
+        '-DUCP_LIBRARY=/usr/lib64/libucp.so '\
+        '-DUCS_LIBRARY=/usr/lib64/libucs.so '\
+        '-DUCT_LIBRARY=/usr/lib64/libuct.so '
+    else:
+        UCX = ""
```

Comment:
should we do explicit NA_USE_UCX=OFF  in this case?

Reply:
that's probably safer depending on the mercury default behavior